crates/bevy_ptr/src/lib.rs

```diff
@@ -155,15 +163,23 @@ impl<'a> PtrMut<'a> {
         self.0.as_ptr()
     }
 }
+impl<'a, T> From<&'a mut T> for PtrMut<'a> {
+    #[inline]
+    fn from(val: &'a mut T) -> Self {
+        // SAFETY: The returned pointer has the same lifetime as the passed reference.
```

Comment:
This needs to document that the reference is both mutable and unaliased for this safety comment to be correct.